test/download-file.spec.js

```diff
@@ -0,0 +1,56 @@
+import expect from 'expect';
+import sinon from 'sinon';
+import fs from 'fs';
+
+import { writeImageOnDisk } from '../src/download-file';
+
+/* eslint-disable  func-names, prefer-arrow-callback */
+describe('writeImageOnDisk', function() {
```

Comment:
Can you please explain a little bit more the problem? I believe you, but I want to understand it for myself. 🙃